study-builder/studies/testboston/snippets/question-live-with-other-people.conf

```diff
@@ -9,9 +9,37 @@
       {
         "name": "live_with_other_people_prompt",
         "translations": [
-          {"language": "en", "text": ${i18n.en.live_with_other_people_prompt}},
-          {"language": "es", "text": ${i18n.es.live_with_other_people_prompt}},
-          {"language": "ht", "text": ${i18n.ht.live_with_other_people_prompt}}
+          { "language": "en", "text": ${i18n.en.live_with_other_people_prompt} },
+          { "language": "es", "text": ${i18n.es.live_with_other_people_prompt} },
+          { "language": "ht", "text": ${i18n.ht.live_with_other_people_prompt} }
+        ]
+      }
+    ]
+  },
+  "trueTemplate": {
+    "templateType": "TEXT",
+    "templateText": "$live_with_other_people_yes",
+    "variables": [
+      {
+        "name": "live_with_other_people_yes",
+        "translations": [
+          { "language": "en", "text": ${i18n.en.option.yes} },
+          { "language": "es", "text": ${i18n.es.option.yes} },
+          { "language": "ht", "text": ${i18n.ht.option.yes} }
+        ]
+      }
+    ]
+  },
+  "falseTemplate": {
+    "templateType": "TEXT",
+    "templateText": "$live_with_other_people_no",
+    "variables": [
+      {
+        "name": "live_with_other_people_no",
+        "translations": [
+          { "language": "en", "text": ${i18n.en.option.no} },
+          { "language": "es", "text": ${i18n.es.option.no} },
+          { "language": "ht", "text": ${i18n.ht.option.no} }
```

Comment:
Would it make more sense to put the yes/no translations in the the snippet that is getting imported:
studies/snippets/bool-question-yes-no.conf

Reply:
Perhaps, but since the import snippet is also imported by other studies, what if the other studies doesn't support the same set of languages that we do?